cosmo_tester/test_suites/cli/linux_test.py

```diff
@@ -102,6 +122,7 @@ def test_cfy_logs_linux_cluster(linux_cluster_cli_tester, tmpdir, logger):
                 [hashlib.md5(open(f.strpath, 'rb').read()).hexdigest() for f
                  in files if 'journalctl' not in f.basename])
             assert set(log_hashes) == set(log_hashes_local)
+            shutil.rmtree(str(tmpdir / 'cloudify'))
```

Comment:
No need to delete it- pytest will deal with that.

Reply:
we do, this is between node logs in the same test. If the folder is dirty the assert will fail